control-plane/pkg/contract/contract.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.15.8
+// 	protoc-gen-go v1.25.0-devel
+// 	protoc        v3.14.0
```

Comment:
I don't have these versions aligned locally, this is what I wanted to avoid by using a container.

/hold

Reply:
Then align them :smile:

Reply:
You can still use a container locally

Reply:
Done! :smile: 

/unhold